components/filecoin/rs/src/signature.rs

```diff
@@ -1,55 +1,29 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
 use crate::message::MessageAPI;
 use blake2b_simd::Params;
 use bls_signatures::Serialize;
+use cid::multihash::MultihashDigest;
 use core::{array::TryFromSliceError, num::ParseIntError};
+use fvm_ipld_encoding::to_vec;
 use fvm_ipld_encoding::Cbor;
+use fvm_ipld_encoding::DAG_CBOR;
+use fvm_shared::address::set_current_network;
+use fvm_shared::address::Network;
 use fvm_shared::address::Protocol;
+use fvm_shared::crypto::signature::{Signature, SignatureType};
```

Comment:
`SignatureType` doesn't seem to be used. Please remove it so the compiler doesn't warn.